src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form1"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    data-qa="big"
+>
+      <input
+        class="input1"
+        type="text"
+        placeholder="`Try &quot;Los Angeles&quot;`"
+        data-qa="keypress"
+     >
+      <svg
+        class="img1">
+      </svg>
+    </form>
+    <form
+    class="form2"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    data-qa="small"
+    >
```

Comment:
```suggestion
    <form
      class="form2"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      data-qa="small"
    >
```
check code style and fix indents everywhere it is needed